src/runTasks.ts

```diff
@@ -1,13 +1,14 @@
 import { info, error, setFailed } from '@actions/core';
 import gatherAllInputs from './tasks/gatherAllInputs';
 import runJest from './tasks/runJest';
-import postComment from './tasks/postComment';
+import postCommentImport from './tasks/postComment';
 
 import packageJson from '../package.json';
 
 const runTasks = async (
     getInputParam?: (key: string) => string,
     execSyncParam?: (command: string) => Buffer,
+    postComment = postCommentImport,
     actuallyPostComment = true
```

Comment:
I like this implementation well enough, but let's actually remove the `actuallyPostComment` boolean.  With the `postComment` now being injected, we can just instead inject something that doesn't post a comment (which I believe you are already doing in all of the tests).

This will likely reduce confusion to future contributors, since it is now redundant.  Hopefully this makes sense, should just be two changes in this function (and making all of the tests not hand in `false` anymore).

Reply:
@dkershner6 Thanks for your review, it makes total sense actually it was indeed confusing to what this boolean is doing as it is not exposed but I thought you might have thought of a use case that you want to handle in the future.
One last thing for this fix is that using user type is a bit tricky I still not able to find what type is the organizational bot is. It can be probably just me being new to github API but I'm still not able to see any specification or schema that lists all possible values of user.type.
I appreciate your help if you have this information even better we can add it to the documentation to help consumers find the right value for their use case, otherwise we can probably use user.login instead of user.type to get the existing comment.
Let me know what you think

Reply:
I dont have any inside information on that field, I agree that GitHub mostly seems to call it a string, and the field doesnt even exist in the graphql API.  For now, I think we can just say `Bot` and `User`, and perhaps another one if you know of one.

Reply:
Indeed same here I couldn't find this information for the organization bot. I read a lot of documntation about account types but sadly still no luck to figure out what the value for the bot an organization would have. Hence my proposal of using user.login instead of user.type then any consumer have access to this information as it is the login name they have created their machine account with WDYT?

Reply:
I am good with that (as I was actually good with getting rid of the check entirely), but let's make the default be "any" (or not checking user at all).

So long as we combine that with my request to delete the param above, I think this is good to go.

Reply:
Oh actually it might be even better to remove this entirely and remove the input then rather than adding one I'm only concerned about if someone would quoted this comment in another comment I'll test and if the check for the comment body alone is enough I believe it would be better to go with it.

Reply:
I tested and as expected removing the check entirely is working well. 
I made the suggested change 👍🏻